llvm/lib/Analysis/TargetLibraryInfo.cpp

```diff
@@ -1503,9 +1509,9 @@ bool TargetLibraryInfoImpl::getLibFunc(const Function &FDecl,
                                        LibFunc &F) const {
   // Intrinsics don't overlap w/libcalls; if our module has a large number of
   // intrinsics, this ends up being an interesting compile time win since we
-  // avoid string normalization and comparison. 
```

Comment:
These changes probably not necessary

Reply:
let's keep it... the trailing white space is bad in source code (my editor automatically deletes it on saving a file)